sdk/storage/azqueue/service_client.go

```diff
@@ -0,0 +1,154 @@
+//go:build go1.18
```

Comment:
Nit: I'd name the file simply `client.go` so it matches the name of the client.  The other clients would have snake-cased file names to match.

Reply:
done